app/scripts/modules/appengine/loadBalancer/details/details.controller.ts

```diff
@@ -65,10 +66,8 @@ class AppengineLoadBalancerDetailsController {
     this.confirmationModalService.confirm({
       header: 'Really delete ' + this.loadBalancer.name + '?',
       buttonText: 'Delete ' + this.loadBalancer.name,
-      provider: 'appengine',
```

Comment:
Same comment above - @danielpeach can you confirm removing this (right now, doesn't need to be in this branch) does not affect the behavior of the modal? Same with `applicationName` below.